redash/handlers/queries.py

```diff
@@ -403,3 +447,60 @@ def get(self):
                 for name, count in tags
             ]
         }
+
+
+class QueryFavoriteListResource(BaseResource):
+    def get(self):
+        search_term = request.args.get('q')
+
+        if search_term:
+            base_query = models.Query.search(search_term, self.current_user.group_ids, include_drafts=True, limit=None)
+            favorites = models.Query.favorites(self.current_user, base_query=base_query)
+        else:
+            favorites = models.Query.favorites(self.current_user)
+
+        favorites = filter_by_tags(favorites, models.Query.tags)
+
+        # order results according to passed order parameter,
+        # special-casing search queries where the database
+        # provides an order by search rank
+        ordered_favorites = order_results(favorites, fallback=bool(search_term))
+
+        page = request.args.get('page', 1, type=int)
+        page_size = request.args.get('page_size', 25, type=int)
+        response = paginate(
+            ordered_favorites,
+            page,
+            page_size,
+            QuerySerializer,
+            with_stats=True,
+            with_last_modified_by=False,
+        )
+
+        self.record_event({
+            'action': 'load_favorites',
+            'object_type': 'query',
+            'params': {
+                'q': search_term,
+                'tags': request.args.getlist('tags'),
+                'page': page
+            }
+        })
+
+        return response
+
+
+class QueryVersionListResource(BaseResource):
+    @require_permission('view_query')
+    def get(self, query_id):
+        results = models.Change.filter(
```

Comment:
This also seems to cause an issue in `tests/handlers/test_queries.py`. Wondering if the intention was `models.Change.query.filter(...)`